### ISSUE TITLE
Add a code of conduct page to the meet up site (copy in ticket)

### ISSUE THREAD

**Issue by gregdorward** [redbadger/website-honestly#573]
Copy:

Code of Conduct

TL;DR — Don't be a Jerk :)

All attendees are expected to adhere to the Code of Conduct at all meetup venues. This includes all React London volunteers and sponsors.

Harassment includes, but is not limited to:

Verbal comments that negatively target a person in regards to their gender, sexual orientation, disability, physical appearance, body size, race, age, or religion.
Continued verbal or physical contact with an attendee who has asked you to stop.
Sexual images in public spaces
Deliberate intimidation, stalking, or following.
Harassing photography or recording.
Sustained disruption of talks or other events.
Inappropriate physical contact.
Unwelcome sexual attention.
Advocating for, or encouraging, any of the above behaviour.

In addition to the above, no weapons or illegal substances are allowed in any meetup venues at any time.

Enforcement

Participants asked to stop any harassing behavior are expected to comply immediately.

If a participant engages in harassing behaviour, event organisers retain the right to take any actions to keep the event a welcoming environment for all participants. This includes warning the offender or expulsion from meetup.

Event organisers may take action to redress anything designed to, or with the clear impact of, disrupting the event or making the environment hostile for any participants.

Reporting

If someone makes you or anyone else feel unsafe or unwelcome, please report it as soon as possible. Harassment and other code of conduct violations reduce the value of our event for everyone.

We want you to be happy at React London events.

Talk to us

In addition to speaking to us in person, you can also contact us using the following methods:

Email us hello@react.london
DM us @reactlondon_

**Comment by chris-brookes** [comment]
This has been completed and deployed live through https://github.com/redbadger/react.london/pull/358
